### ISSUE TITLE
Program Freezes as soon as camera is plugged in and turned on MacOS Sonoma 14.1.2

### ISSUE THREAD

**Issue by BensFunWorld** [tahoma2d/tahoma2d#1414]
### Bug Description

When trying to use the stop motion portion of Tahoma2D the program will freeze when I plug in the camera and turn it on. Program will work with built-in FaceTime camera but freezes as soon as DSLR is plugged in.
When attempting to open program with DSLR already plugged in and turned on the loading splash screen gets stuck at Creating New Window
Using Canon EOS Rebel T6 (1300D)
MacBook Pro M1 16GB

### Steps to reproduce

1. Tahoma2D
2. Plug in DSLR
3. Turn DSLR on
4. Tahoma2D freezes

### Software Version

1.4.1

### Operating System

macOS Sonoma 14.1.2

### Graphics Card

M1

**Comment by artisteacher** [comment]
I don't have the same camera, but when I'm in school tomorrow, I'll see what results I get with a Canon SL2.

There probably won't be a difference, but just in case, have you tried a recent nightly build?

**Comment by artisteacher** [comment]
So on an iMac running Ventura, I can successfully use the SL2. T2D opens just fine whether the camera is connected or not. I do see the camera listed twice, once as an SL2 (probably the canon sdk) and a second time as a dp200 (an alternate name, probably libgphoto2). The sl2 starts live view and works as expected. The dp200 fails to start live view.

**Comment by artisteacher** [comment]
One last thing, I'm using a nightly build from Feb. 1st, in case that's had an impact.

**Comment by manongjohn** [comment]
We're currently on Canon EDSDK version 13.17.10.  The latest version is 13.17.12 and neither .11 or .12 mention any changes made to support Sonoma.  The Rebel T6 is a supported camera for the Canon EDSDK version we are on.

Problem it could be an issue with libgphoto2 trying to connect to the camera and is somehow stuck.  The T6 is supported and is in the current version of our ligphoto2 library.   Going to be hard to figure out what's going on.

**Comment by BensFunWorld** [comment]
Discovered a solution. My camera didn't have an SD card in the slot. I notice the issues wasn't exclusive to Tahoma and even EOS Webcam Utility wasn't able to recognize the camera until the SD card was inserted. Oversight on my part and likely isn't related to Tahoma2D, seeing how it didn't even work with Canon software. 
When I did get the camera to work it was important to use the profile with the correct model. The T6 and 1300D are technically the same but my camera is a Rebel T6 and while the option for the 1300D could be selected it would only work when I choose the Rebel T6 model.
Aside from live view occasionally freezing (capture still worked and sometimes taking a picture would unfreeze live view) everything worked great. Closing since issue was resolved and was not related to Tahoma2D.
